ui/src/app/edge/live/common/storage/storage.component.html

```diff
@@ -0,0 +1,23 @@
+<ng-container *ngIf="isInitialized">
+    <oe-flat-widget [img]="storageItem" [title]="'General.storageSystem'| translate" (click)="presentModal()">
+        <ng-container *ngFor="let component of essComponents, let i = index">
+            <ng-container *ngIf="essComponents.length > 1">
+                <oe-flat-widget-line [name]="component.alias">
+                </oe-flat-widget-line>
+            </ng-container>
+            <ng-container *ngIf="checkStateOfCharge(component.id)">
```

Comment:
Methodenaufrufe aus Templates sind immer schlecht - siehe https://medium.com/showpad-engineering/why-you-should-never-use-function-calls-in-angular-template-expressions-e1a50f9c0496

Warum soll der Speicher ausgeblendet werden, wenn es keinen SoC gibt?

Reply:
das war auch bisher so. kann man natürlich noch ändern

Reply:
checkStateOfCharge(component.id) wird entfernt